x-pack/plugins/reporting/server/routes/common/jobs/jobs_query.ts

```diff
@@ -81,7 +81,6 @@ export function jobsQueryFactory(reportingCore: ReportingCore): JobsQueryFactory
             filter: {
               bool: {
                 must: [
-                  { terms: { jobtype: jobTypes } },
```

Comment:
This is a bug that prevents users from seeing past reports, which are no longer supported in implementation, in the listing